src/rascal/structure_managers/structure_manager_lammps.hh

```diff
@@ -180,6 +182,8 @@ namespace rascal {
      */
     size_t get_nb_clusters(int order) const;
 
+    bool are_any_centers_masked() const { return false; }
```

Comment:
As a side note: is the masking only for training?

Reply:
the masking is for properties that are different from the energy like transfert integral between dimer molecules. So both for training and predictions.